### DISCUSSION TITLE
翻译字幕应该用什么模型？

### DISCUSSION THREAD

**Discussion by ovo-Tim** [ymcui/Chinese-LLaMA-Alpaca-2#414] [Q&A] [answered]
是基座模型 LlaMA 还是 Alpaca？
另外是否需要长上下文？
谢谢😃

**Comment by ovo-Tim** [comment]
我想我应该用 长上下文的基座模型🧐

**Comment by ymcui** [comment, marked as answer]
只要是涉及实际下游应用任务的都应该使用Alpaca系列。
不过你为什么不用一个专用的翻译模型呢？

**Reply by ovo-Tim** [reply]
感谢指导🙏
关于专门的翻译模型的话，一是没有找到好的模型(欢迎推荐)；二是我觉得用大语言模型翻译可以结合上下文，翻译准确一些，而且可以实现一些特殊功能比如技术名词后面附上英文原文。😉

**Comment by greatiliad** [comment]
去看看whisper

**Reply by ovo-Tim** [reply]
Whisper 已经在用了，即使是 Small 模型效果也相当好。还需要模型来翻译字幕

**Comment by ovo-Tim** [comment]
实测翻译效果不是很好
![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/assets/116942368/6450bc19-e100-4d00-8471-1fa8e490eec8)
